src/config/dynamic-theme-fixes.config

```diff
@@ -14532,6 +14532,18 @@ li.level0 > a:hover {
 
 ================================
 
+pcgamer.com
+
+CSS
+img {
+	mix-blend-mode: normal !important;
+}
+img[alt="Best Buy"], [alt="Amazon"] {
```

Comment:
Where are these elements?

Reply:
They're in all of the hardware buying guides on the site, all the articles such as "best * of 2022". You can find a list of them here: https://www.pcgamer.com/hardware/buying-guides/
This includes the example URL I included above.

Reply:
I fail to find this element:
![image](https://user-images.githubusercontent.com/25481501/189545892-7c69369b-eb09-4327-87ab-76609f5cae7c.png)

Reply:
You can find both elements on pages such as 
https://www.pcgamer.com/best-gaming-chairs/
https://www.pcgamer.com/best-freesync-monitor/
https://www.pcgamer.com/best-light-gaming-mouse/
https://www.pcgamer.com/best-computer-speakers/
https://www.pcgamer.com/the-best-gaming-mouse/

Reply:
Still nothing...
![image](https://user-images.githubusercontent.com/25481501/190004208-608e94a9-073e-4294-b63c-20f64aa3e3dd.png)

Might be a region-locked featured.

Reply:
Could you post a before and after screenshot?

Reply:
Sure, so this is how a page looks in the U.S.:
![Screen Shot 2022-09-17 at 02 43 08](https://user-images.githubusercontent.com/91507075/190845734-ac44ea5c-21a4-4266-8b6c-3975ef4f7d61.png)

This is how it looks in Germany:
![Screen Shot 2022-09-17 at 02 44 31](https://user-images.githubusercontent.com/91507075/190845785-0553b486-61a5-4213-89a7-ae65c8d3ea3d.png)

The idea behind mix-blend-mode:difference was because of this issue:
![Screen Shot 2022-09-17 at 02 42 35](https://user-images.githubusercontent.com/91507075/190845546-2259792b-544e-45f1-b63f-fe17cb930b6b.png)

Considering Best Buy only sells exclusively in the U.S. it's not really that important to keep the selector for it.

Reply:
Hmm, weird geo-lock. Please use `mix-blend-mode: normal` and add the selectors to the `INVERT` section.